Applications/Spire/Source/Login/LoginWindow.cpp

```diff
@@ -33,6 +34,18 @@ namespace {
       set(Fill(QColor::fromRgb(0xE6, 0x3F, 0x45)));
     return style;
   }
+
+  auto INPUT_STYLE(StyleSheet style) {
+    style.get(Any()).set(border(scale_width(0), QColor::fromRgb(0, 0, 0, 0)));
+    auto font_style = Styles::find<TextStyle>(style.find(Any())->get_block());
+    if(font_style) {
```

Comment:
Can you describe the intention here? Why not unconditionally set the font?

Reply:
I was thinking it's possible that the StyleSheet may not have a TextStyle set. That's not particularly likely because this is the TextBox's stylesheet, but it seemed like good practice to handle that case.

Reply:
Oh I see what you mean. Go ahead and fully set the font, including ```"Roboto"``` etc...

Reply:
Actually it's fine, leave it as is.